style2.css

```diff
@@ -400,4 +452,459 @@
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.gitlab {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+}
+
+.term {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 120px;
+}
+
+.projects {
+  width: 327px;
+  height: 363px;
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 25px;
+  white-space: initial;
+  margin-left: 40px;
+  margin-top: 20px;
+}
+
+.name {
+  height: 48px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5130px;
+}
+
+.name-last {
+  height: 48px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5200px;
+}
+
+.enter {
+  height: 114px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5270px;
+}
+
+.name:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.name-last:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.enter:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.touch-btn {
+  background-color: #36b37c;
+  color: #ebf0ee;
+  font-size: 17px;
+  border-radius: 5px;
+  height: 48px;
+  width: 129px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  position: absolute;
+  top: 5480px;
+  left: 80px;
+}
+
+.touch-btn:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+  cursor: pointer;
+}
+
+.icon-row2 {
+  display: flex;
+  position: absolute;
+  top: 5580px;
+  left: 78px;
+}
+
+.email-address {
+  width: 375px;
+  height: 189px;
+}
+
+.email {
+  color: #42526e;
+  font-size: 20px;
+  line-height: 32px;
+  position: absolute;
+  left: 70px;
+  top: 5620px;
+}
+
+.email:hover {
+  text-decoration: underline rgb(6, 14, 243);
+}
+
+.top-row li {
+  display: inline-block;
+}
+
+.bottom-row li {
+  display: inline-block;
+}
+
+.center-row li {
+  display: inline-block;
+  margin-bottom: 20px;
+}
+
+.contactpage {
+  display: flex;
+  flex-direction: column;
+}
+
+#user_name {
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+}
+
+#text-box {
+  width: 327px;
+  height: 114px;
+}
```

Comment:
- [ ] [REQUIRED] Please add correct color `#091E42;` to your `#text-box{...}` 

![image](https://user-images.githubusercontent.com/63915024/186476466-e5ae43de-86f8-4f78-9413-bad2d8ee199d.png)

Reply:
ok changed